examples/build-package/main.tf

```diff
@@ -17,17 +17,50 @@ resource "random_pet" "this" {
 # Build packages
 #################
 
-# Create zip-archive of a single directory where "pip install" will also be executed (default for python runtime)
+# Create zip-archive of a single directory where "pip install" will also be executed (default for python runtime with requirements.txt present)
 module "package_dir" {
   source = "../../"
 
   create_function = false
 
-  runtime     = "python3.8"
-  source_path = "${path.module}/../fixtures/python3.8-app1"
+  build_in_docker = true
+  runtime         = "python3.8"
+  source_path     = "${path.module}/../fixtures/python3.8-app1"
+  artifacts_dir   = "${path.root}/builds/package_dir/"
```

Comment:
Specifying different `artifacts_dir` for each module makes it easier to check lambda packages content.